ams/ams-server/src/main/java/com/netease/arctic/ams/server/mapper/FileInfoCacheMapper.java

```diff
@@ -60,8 +62,8 @@ public interface FileInfoCacheMapper {
   @Select(
         "select add_snapshot_id, count(1) as cnt, sum(file_size) as size, commit_time from " + TABLE_NAME + " where " +
             "table_identifier = #{tableIdentifier, typeHandler=com.netease.arctic.ams.server.mybatis" +
-            ".TableIdentifier2StringConverter} and action!='replace' group by add_snapshot_id, commit_time order by " +
-            "commit_time desc")
+            ".TableIdentifier2StringConverter} and producer!='OPTIMIZE' group by add_snapshot_id, commit_time order " +
```

Comment:
Put the 'OPTIMIZE' as param to function  will be clearer?

Reply:
maybe should rename the function? just like named `getTxExcludeOptimize` because there is show all transactions exclude optimize transaction, but not show one type transaction. let me konw what do you think